main.go

```diff
@@ -494,6 +494,9 @@ func decodeGetRoutes(payload []byte) string {
 
 	for _, routeMapCell := range getRoutesOutProto.GetRouteMapCell() {
 		for _, route := range routeMapCell.GetRoute() {
+			if route.RouteSubmissionStatus.Status != pogo.RouteSubmissionStatus_PUBLISHED {
+				continue
```

Comment:
I tested this change and added an additional log line on line 498... we never get into the "Route not Published".... even I have a route which is not published yet. so that change does not skip processing unpublished routes

Reply:
Hi Niantic!